packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -81,7 +81,8 @@ function NotesSidebarContent( {
 }
 
 function NotesSidebar( { postId, mode } ) {
-	const [ showCommentBoard, setShowCommentBoard ] = useState( false );
+	// Enum: 'closed' | 'creating' | 'open'
+	const [ showCommentBoard, setShowCommentBoard ] = useState( 'closed' );
```

Comment:
We should probably rename this into something more appropicate.

```js
const [ newNoteFormState, setNewNoteFormState ] = useState( 'closed' );
```

Reply:
Should I rename the state and setter before we merge this, or should we keep the old naming for now? WDYT?